library/pom.xml

```diff
@@ -378,6 +378,11 @@
         </exclusion>
       </exclusions> 
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-hdfs</artifactId>
+         <version>2.6.1</version>
+      </dependency>
```

Comment:
will this copy hadoo-hdfs jar in app package? see if it can be made provided scope.